### PR TITLE
Add Apple CryptoKit

### PR DESCRIPTION
**Pull request by cak**
Add [Apple CryptoKit](https://developer.apple.com/documentation/cryptokit/)

Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [X] Your contribution is written in the 2nd person (e.g. you)
- [X] Your contribution is written in an active present form for as much as possible.
- [X] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [X] Your contribution has proper formatted markdown and/or code
- [X] Any references to website have been formatted as [TEXT](URL “NAME”)
- [X] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1317 and #961.

### CODE REVIEW

**Comment by commjoen** [comment]
@DarkInnocence can you review this pr please :)?

File: Document/0x06e-Testing-Cryptography.md [hunk 1]
```diff
@@ -8,6 +8,57 @@ In the "Cryptography for Mobile Apps" chapter, we introduced general cryptograph
 
 Apple provides libraries that include implementations of most common cryptographic algorithms. [Apple's Cryptographic Services Guide](https://developer.apple.com/library/content/documentation/Security/Conceptual/cryptoservices/GeneralPurposeCrypto/GeneralPurposeCrypto.html "Apple Cryptographic Services Guide") is a great reference. It contains generalized documentation of how to use standard libraries to initialize and use cryptographic primitives, information that is useful for source code analysis.
 
+##### CryptoKit
+
+Apple CryptoKit was released with iOS 13 and is built on top of Apple's native cryptographic library `corecrypto`. The Swift framework provides a strongly typed API interface, has effective memory management, conforms to equatable, and supports generics. CryptoKit contains secure algorithms for hashing, symmetric-key cryptography, and public-key cryptography. The framework can also utilize the hardware based key manager the Secure Enclave.
```
Comment (TheDauntless) [inline Document/0x06e-Testing-Cryptography.md]:
```suggestion
Apple CryptoKit was released with iOS 13 and is built on top of Apple's native cryptographic library `corecrypto`. The Swift framework provides a strongly typed API interface, has effective memory management, conforms to equatable, and supports generics. CryptoKit contains secure algorithms for hashing, symmetric-key cryptography, and public-key cryptography. The framework can also utilize the hardware based key manager from the Secure Enclave.
```

File: Document/0x06e-Testing-Cryptography.md [hunk 2]
```diff
@@ -8,6 +8,57 @@ In the "Cryptography for Mobile Apps" chapter, we introduced general cryptograph
...
+let textHash = String(describing: hash)
+print(textHash) // Print hash text
+```
+
+For more information about Apple CryptoKit, please visit the following resources:
+
+- [Apple CryptoKit | Apple Developer Documentation](https://developer.apple.com/documentation/cryptokit)
+- [Performing Common Cryptographic Operations | Apple Developer Documentation](https://developer.apple.com/documentation/cryptokit/performing_common_cryptographic_operations)
+- [WWDC 2019 session 709: Cryptography and Your Apps](https://developer.apple.com/videos/play/wwdc19/709/)
+- [How to calculate the SHA hash of a String or Data instance - Hacking with Swift](https://www.hackingwithswift.com/example-code/cryptokit/how-to-calculate-the-sha-hash-of-a-string-or-data-instance)
```
Comment (TheDauntless) [inline Document/0x06e-Testing-Cryptography.md]: Please add alt-texts for the urls: `[TEXT](http://url "alt text")`

Comment (cak) [reply Document/0x06e-Testing-Cryptography.md]: Sorry, I missed that - fixed with dd0611be4e98ab53fffc13ae363b0cf26a752e50.

**Pull request merged**